docs/operation/upgrade.rst

```diff
@@ -9,8 +9,8 @@ Supported Versions
 ******************
 .. note::
 
-    MetalK8 supports upgrade **strictly** from one supported
-    minor version to another. For example:
+    MetalK8s supports upgrade **strictly** from one minor version to another.
```

Comment:
Not sure about this wording... @lucieleonard thoughts?

Maybe `from one minor version to the next`? Or `supports upgrade of **at most** one minor version at a time`?

Reply:
I like the second one, I'll go for it.